plugins/foreman_virt_who_configure/0.5/index.md

```diff
@@ -0,0 +1,272 @@
+
+## 1.2 Supported Hypervisors:
+
+* oVirt
+* Libvirt
+* VMware
+* Hyper-V
+* Kubevirt
+* Nutanix AHV
+
+# 2. Installation
+
+This plugins requires [Katello](https://theforeman.org/plugins/katello/index.html) to be installed in your Foreman instance. If you have it, simply install a package with the plugin (rpm only), run the database migrations and seed by running:
+
+    yum install rubygem-foreman_virt_who_configure
+    foreman-rake db:migrate
+    foreman-rake db:seed
+
+The plugin package lives at [Katello repositories](http://yum.theforeman.org/plugins/).
+
+If Foreman was installed from git, you can simply add foreman-virt_who_configure gem as one of the dependencies. Don't forget to run the database migrations and seed afterwards and restart the Foreman services. You still need to have Katello installed.
+
+Note for Satellite 6 users, the plugin is installed out of the box and no extra action is required.
+
+# 3. Usage (features description)
+
+The plugin adds a new link __Virt-who configurations__ under the __Infrastructure__ menu. This page lists all virt-who configurations that were created in Foreman. 
+
+To create a new configuration, click the __Create Config__ button. Once you provide all the information, the configuration is registered and you can deploy it. Each configuration provides a configuration script that you will find after clicking on it's name. The configuration script should be run on a host where you want to deploy the virt-who instance at, e.g. the same host where Foreman runs. 
+
+Existing virt-who configurations can be deleted when no longer required. The Plugin also provides a monitoring widget for the dashboard. Please see chapters below for more details on every feature.
+ 
+## 3.1 Creating
+
+In order to create a new virt-who configuration, navigate to __Infrastructure__ -> __Virt-who configurations__ and click the __Create Config__ button. This takes you to a form which guides you through what we need to know in order to successfully configure virt-who. Most of the fields provide some inline help, please click the help icon for more information.
+
+Note that every virt-who configuration must exist in an organization. If you haven't selected any organization, you'll see Owner field in the form where you will need to choose an Organization. If you already are in some organization context, the Owner field is not displayed and the virt-who configuration will be automatically created in this Organization scope.
+  
+Default values which are prefilled are recommended values which are usually not needed to be changed. This applies to especially to Interval, Hypervisor ID and Satellite server FQDN. The Satellite server FQDN is where the virt-who instance will send reports to so make sure this FQDN is reachable from the host, where the virt-who instance will run. The default value is determined based on what setting `foreman_url` is configured to.
```

Comment:
```suggestion
Values which are prefilled are default recommended values, which usually don't need any changes. This applies especially to Interval, Hypervisor ID and Satellite server FQDN. The Satellite server FQDN is where the virt-who instance will send reports to so make sure this FQDN is reachable from the host, where the virt-who instance will run. The default value is determined based on what setting `foreman_url` is configured to.
```
Should this reference be to Foreman instead of Satellite?